requirements.yml

```diff
@@ -0,0 +1,2 @@
+---
+- src: geerlingguy.docker
```

Comment:
Do we need to run by Security that we're pulling this in?

Reply:
Don't think so. It's MIT license, and just downloads the docker library files.
https://github.com/geerlingguy/ansible-role-docker
The previous code we're replacing is a call to a remote script that does the downloading and installation, this is possibly better as it's more explicit as to what it's doing.
I've added @rigoford to the review so he's aware.